# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/mpeghdec/remove-submodules.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- mpeghdec-r3.0.1/demo/CMakeLists.txt.vanilla	2025-09-24 19:20:17.232614856 +0200
+++ mpeghdec-r3.0.1/demo/CMakeLists.txt	2025-09-24 19:20:19.606132046 +0200
@@ -1,5 +1,3 @@
-FetchContent_MakeAvailable(ilo mmtisobmff)
-
 if(${mpeghdec_BUILD_DECODER})
   add_executable(mpeghDecoder "mpeghDecoder/main_mpeghDecoder.cpp")
   target_link_libraries(mpeghDecoder mpeghdec mmtisobmff ilo)
--- mpeghdec-r3.0.1/CMakeLists.txt.vanilla	2025-09-24 19:20:04.769373042 +0200
+++ mpeghdec-r3.0.1/CMakeLists.txt	2025-09-24 19:20:12.779524984 +0200
@@ -14,22 +14,6 @@
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 
-# Define dependencies
-include(FetchContent)
-
-# ilo from 2023-05-30
-FetchContent_Declare(
-  ilo
-  GIT_REPOSITORY https://github.com/Fraunhofer-IIS/ilo.git
-  GIT_TAG        r1.0.0
-)
-# mmtisobmff from 2023-05-30
-FetchContent_Declare(
-  mmtisobmff
-  GIT_REPOSITORY https://github.com/Fraunhofer-IIS/mmtisobmff.git
-  GIT_TAG        r1.0.1
-)
-
 get_directory_property(parentDir PARENT_DIRECTORY)
 
 # Set either of the two variables to OFF in case not both functionalities are required:
